test/unit/utils/dom.test.js

```diff
@@ -2,6 +2,8 @@ import document from 'global/document';
 import * as Dom from '../../../src/js/utils/dom.js';
 import TestHelpers from '../test-helpers.js';
 
+q.module('dom');
```

Comment:
makes the output make more sense. Since I updated qunit to 1.18 there was an error that looked like it was in text tracks but was actually in url.